JS/plugin/Print.js

```diff
@@ -0,0 +1,7 @@
+class Print {
```

Comment:
1) What is the point of creating a separate class for converting objects into strings? It can be just a funstion.
2) Please give a more meaningful name to your function.